api/database/factories/PoolFactory.php

```diff
@@ -206,4 +208,19 @@ public function withAssessments()
             $step2->poolSkills()->sync(array_slice($poolSkillArray, 5, 5, true));
         });
     }
+
+    /** Update the pool's pool_skill table to be complete
+     * For use in testing and seeding convenience, edits pool skill models after the pivot table was filled
+     *
+     * @return \Illuminate\Database\Eloquent\Factories\Factory
+     */
+    public function withCompletePoolSkills()
+    {
+        return $this->afterCreating(function (Pool $pool) {
+            PoolSkill::where('pool_id', $pool->id)
+                ->update([
+                    'required_skill_level' => $this->faker->randomElement(array_column(SkillLevel::cases(), 'name')),
```

Comment:
You should add skill levels in the same place the PoolSkills are created, instead of optionally after the fact.

Reply:
I tried to have it in `configure()` but that just wasn't working for me 😖